### ISSUE TITLE
--no-devtool doesn't work

### ISSUE THREAD

**Issue by snoack** [webpack/webpack-cli#2175]
**Describe the bug**

According to the [documentation](https://webpack.js.org/api/cli/#negated-flags) there is a `--no-devtool` flag. However, if povided on the command line `webpack` fails with an unknown argument error.

**What is the current behavior?** / **To Reproduce**

```
$ webpack --no-devtool
[webpack-cli] Unknown argument: --no-devtool
```

**Expected behavior**

`webpack --no-devtool` should complete successfully, while not generating any source maps.

**Comment by jamesgeorge007** [comment]
It will be available in the next release #2132.
cc @alexander-akait

**Comment by alexander-akait** [comment]
Yes, it will be available in the next release
